Gopkg.toml

```diff
@@ -1,29 +1,43 @@
+# Packages should only be added to the following list when we use them *outside* of our go code.
+# (e.g. we want to build the binary to invoke as part of the build process, such as in
+# generate-proto.sh). Normal use of golang packages should be added via `dep ensure`, and pinned
+# with a [[constraint]] or [[override]] when version is important.
 required = [
+  "github.com/golang/protobuf/protoc-gen-go",
   "github.com/gogo/protobuf/protoc-gen-gofast",
   "github.com/gogo/protobuf/protoc-gen-gogofast",
-  "golang.org/x/sync/errgroup",
   "k8s.io/code-generator/cmd/go-to-protobuf",
   "github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway",
   "github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger",
-  "github.com/golang/protobuf/protoc-gen-go",
-  "golang.org/x/tools/cmd/cover",
```

Comment:
Our use of [cover](https://golang.org/cmd/cover/) for coverage testing is, to my understanding, outside of compilation, as we run it with a command that looks like `go test -v -covermode=count -coverprofile=coverage.out ...`. It's interesting to me that the tests all ran successfully without it being installed from this list. How useful is it to keep in the list?

Reply:
That command does not make use of `golang.org/x/tools/cmd/cover`, it's using the built in coverage available in `go test`. `golang.org/x/tools/cmd/cover` has been part of the standard library since 1.5, so  the import of this package was never necessary.

Reply:
Fancy! Thanks, @jessesuen.